pkg/cmd/dev/build.go

```diff
@@ -453,11 +457,21 @@ func (d *dev) getBasicBuildArgs(
 	return args, buildTargets, nil
 }
 
-// Given a list of Bazel arguments, find the ones starting with --config= and
-// return them.
+// Given a list of Bazel arguments, find the ones that represent a "config"
+// (either --config or -c) and return all of these. This is used to find
+// the appropriate bazel-bin for any invocation.
 func getConfigArgs(args []string) (ret []string) {
+	var addNext bool
 	for _, arg := range args {
-		if strings.HasPrefix(arg, "--config=") {
+		if addNext {
+			ret = append(ret, arg)
+			addNext = false
+		}
+		if arg == "--config" || arg == "--compilation_mode" || arg == "-c" {
+			ret = append(ret, arg)
+			addNext = true
```

Comment:
A nit. You can add `continue` here, so you don't even try to evaluate the next `if` (and potential future ifs).

Reply:
I used `if .. else if` for this to accomplish the same thing without using `continue`.

Reply:
Oh, right. nm then!